README.md

```diff
@@ -0,0 +1,26 @@
+# kratky_rg_izero
+Create overlaid Kratky plots from a directory of SAXS data from [PRIMUS analysis software](https://www.embl-hamburg.de/biosaxs/primus.html),
+using RG and I(0) values.
+
+## Requirements
+Python 3 (created using Python 3.8)
+
+Libraries: matplotlib, pandas
+
+## Usage
+Intended for use with .out files produced by PRIMUS.
+
+Place all files for a single Kratky plot in one directory with no other .out files.
+
+Run from command line examples:
+
+`python3 rg_and_io.py path/to/datafiles -o save/to/dir/plotname -c red blue`
+
+`python3 rg_and_io.py user1/saxs/exp_1 -o user1/saxs/exp_1/exp_1_kratky -c red blue`
```

Comment:
I went back and forth on this. I wasn't assuming whoever is using this will know much about extensions, command line, or matplotlib, based on experiences J has related. Maybe an overridable default.